assets/js/blocks/cart-checkout/mini-cart/block.tsx

```diff
@@ -193,10 +192,8 @@ const MiniCartBlock = ( {
 				<div
 					className="wc-block-mini-cart__template-part"
 					ref={ contentsRef }
-				>
-					{ /* @todo The `div` wrapper of RawHTML isn't removed on the front end. */ }
-					<RawHTML>{ contents }</RawHTML>
-				</div>
+					dangerouslySetInnerHTML={ { __html: contents } }
```

Comment:
How do we know this is safe, can we ensure that `contents` contains nothing malicious?

Reply:
![image](https://user-images.githubusercontent.com/4463174/151947209-5fc7bb7d-d64e-4c3a-ad96-6a99406cfa56.png)

This is the source code of `RawHTML`, there isn't any check about checking the sanity of the content.

This content is loaded from the `templates` folder and [parsed from the backend](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/66e61cd502fd83d28338b3fcfec340c18fb8fded/src/Utils/BlockTemplateUtils.php#L171).

So I guess it is a pretty safe approach. What do you think?

Reply:
Hey! Thanks for clarifying. Yep it seems fine, the concern of adding malicious scripts in a template is pretty low as a merchant would have to modify their template files, something we can't really stop.

LGTM 👍🏼

Reply:
Thanks!